.github/workflows/h5bench-hdf5-develop.yml

```diff
@@ -416,6 +416,17 @@ jobs:
           cd build-sync
           ./h5bench --debug --abort-on-failure ../samples/sync-macsio.json
 
+      - name: Test h5bench SYNC dlio
+        run: |
+          export HDF5_HOME=/opt/hdf5
+          export ABT_HOME=/opt/argobots
+          export ASYNC_HOME=/opt/vol-async
+
+          export LD_LIBRARY_PATH=$ASYNC_HOME/lib:$HDF5_HOME/lib:$ABT_HOME/lib:$LD_LIBRARY_PATH
```

Comment:
since this benchmark does not have support for ASYNC VOL, please remove those dependencies from here (i.e., async, abt)